source/docs/hardware-components/kit-and-hardware-guide/gobilda.rst

```diff
@@ -29,13 +29,22 @@ Notable components
 
    goBILDA :term:`U-channel <Channel>` (48x48 mm)
 
-- In addition, low-side channel (smaller profile :term:`channel <Channel>`) complements full size channel. Low side channel is just 12mm in height and allows for more compact builds. Additionally, low side channel opens up possibilities such as a kit-based :term:`parallel plate drivetrain <Parallel Plate Drivetrain>`. Low side channel also forms the basis of the cascading kit with a 2 or 4 stage option.
+- Low-side channel (smaller profile :term:`channel <Channel>`) complements full size channel. Low side channel is just 12mm in height and allows for more compact builds. Additionally, low side channel opens up possibilities such as a kit-based :term:`parallel plate drivetrain <Parallel Plate Drivetrain>`. Low side channel also forms the basis of the cascading kit with a 2 or 4 stage option.
 
 .. figure:: images/gobilda/gobilda_low_channel.jpg
    :alt: goBILDA low-side channel
 
    goBILDA low-side channel (12x48 mm)
 
+- A new addition is Mini Low-side channel (even smaller profile :term:`channel <Channel>`) that compliments low-side channel. Mini Low-side channel is 12mm tall and just 32mm wide, making it great for small structures.
```

Comment:
Is it "Low-side" or "low-side" or something else? goBILDA marketing sats "Low-Side"

Reply:
Updating to Low-Side. I should just defer to gobilda in general for this honestly, but I wrote it while watching the low side reveal stream